### PR TITLE
Fixed Donations page loading text & small UI tweaks, seed donations for new setups (#116)

### PR DESCRIPTION
**Pull request by GeniJaho**
- Haven't compiled assets because I'm on Windows and that has sometimes caused me problems
- Not sure about the Vue implementation with loading state. It's straightforward and works, but there might be a better way to do it.

Put everything in a container div 
image

And prevented the image on mobile from stretching. 
image

### CODE REVIEW

File: resources/js/components/DonateButtons.vue [hunk 1]
```diff
@@ -46,8 +46,12 @@ export default {
...
 	{
 		this.loading = true;
-		await this.$store.dispatch('GET_DONATION_AMOUNTS');
-		this.loading = false;
+		await this.$store.dispatch('GET_DONATION_AMOUNTS')
+            .then(() => {
+                this.loading = false;
+
+                this.$emit('donations-loaded')
+            });
```
Comment (xlcrr) [inline resources/js/components/DonateButtons.vue:54]: No need to call `then` here, it won't run until after await anyway

**Review by xlcrr — APPROVED** [review]
Need to test this before merge

File: resources/js/components/DonateButtons.vue (hunk 1 above)
Comment (GeniJaho) [reply resources/js/components/DonateButtons.vue]: True, missed that. Added a commit for it.

**Comment by GeniJaho** [comment]
Any update on this one? I can remove the Vue changes if they feel to risky.

**Pull request merged**